src/platforms/javascript/common/configuration/integrations/default.mdx

```diff
@@ -87,6 +87,21 @@ Available options:
 }
 ```
 
+Here is a code example of how this could be implemented:
+
+```
+document.querySelector('#teststuff').addEventListener('click', () => {
+    e1 = new Error('fooooo');
+    e2 = new Error('bar?');
+    e2.cause = e1;
+    Sentry.captureException(e2);
+  });
+```
+
+<img width="928" alt="Screen Shot 2021-04-21 at 11 34 50 PM" src="https://user-images.githubusercontent.com/65051899/115667031-2d45b980-a2fa-11eb-8689-b6d6011a67f6.png">
```

Comment:
thanks for adding this content. Looks like the image doesn't use our guidelines, which I'll forward to you on Slack. The basics (for future reference):
![text_for_image_if_not_displayed](image_name.png)